rollup.config.js

```diff
@@ -60,6 +61,47 @@ export default (cliArgs) => {
                 commonjs(),
                 typescript()
             ],
+        },
+        {
+            input: "src/background-loader.ts",
+            output: {
+                file: "dist/background-loader.js",
+                sourcemap: (isDevMode(cliArgs) || isEmulatorMode(cliArgs)) ? "inline" : false,
+            },
+            plugins: [
+                replace({
+                    preventAssignment: true,
+                    // In Developer Mode, the study does not submit data and
+                    // gracefully handles communication errors with the Core
+                    // Add-on.
+                    __ENABLE_DEVELOPER_MODE__: isDevMode(cliArgs),
+                    __ENABLE_EMULATOR_MODE__: isEmulatorMode(cliArgs),
+                }),
+                webScienceRollupPlugin(),
+                resolve({
+                    browser: true,
+                }),
+                commonjs(),
+                typescript(),
+                copy({
+                    targets: [{
+                        src: [
+                            "node_modules/@mozilla/rally/dist/rally-content.js",
+                        ],
+                        dest: "dist/",
+                    }],
+                    flatten: true,
+                }),
+                copy({
+                    targets: [{
+                        src: [
+                            "node_modules/webextension-polyfill/dist/browser-polyfill.js",
```

Comment:
We should just include it in the SDK as import and not bother including it in individual extensions. What do you think?

Reply:
I agree, there was resistance from the glean.js folks though. I think we should revisit this with them, as I think it would both be nicer for development and also produce a smaller overall bundle if glean were to import the browser-polyfill.

Reply:
For the SDK and web-science we can do this yes.